src/components/LoanCards/GridLoanCard.vue

```diff
@@ -49,6 +49,8 @@
 					:amount-left="amountLeft"
 					:show-now="!enableFiveDollarsNotes"
 					:enable-five-dollars-notes="enableFiveDollarsNotes"
+					:enable-huge-amount="enableHugeAmount"
+					:is-visitor="isVisitor"
```

Comment:
Same question here, why was `isVisitor` added here?

Reply:
In order to generate the huge amount price array for both conditions, logged in users and experiment flag.